Worktez/worktez
# fix(sprint filter):board page sprint filter

### Functionality:
fixing board page sprint filter

### Solution:
changes in board components

### Risk level:

### How to test:
on board page click on filter the sprint name should change to Backlog when sprint number is 0 or -1.
and it should change to Deleted when sprint number is -2 or less
